### PR TITLE
fix: Added version to eks.Cluster with 1.17 as version to be compatible with cdk v1.55.0

### PR DESCRIPTION
**Pull request by dnmach**
…th cdk v1.55.0

Fixes # 

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

**Review by NGL321 — DISMISSED** [review]
Thanks for the fix, and sorry for the delay in review!

😸 😷

File: typescript/eks/cluster/index.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ class EKSCluster extends cdk.Stack {
     });
 
     const eksCluster = new eks.Cluster(this, 'Cluster', {
+      version: eks.KubernetesVersion.V1_17,
```
Comment (shivlaks) [inline typescript/eks/cluster/index.ts:19]: there's a version key on line 22 that should have it's value updated instead of introducing a duplicate key

**Review by shivlaks — CHANGES_REQUESTED** [review]

File: typescript/eks/cluster/index.ts [hunk 2]
```diff
@@ -16,6 +16,7 @@ class EKSCluster extends cdk.Stack {
...
       vpc: vpc,
       defaultCapacity: 0,  // we want to manage capacity our selves
       version: eks.KubernetesVersion.V1_16,
```
Comment (ryparker) [inline typescript/eks/cluster/index.ts:22]:
```suggestion
```
This is a duplicate key.

**Review by ryparker — CHANGES_REQUESTED** [review]

**Comment by ryparker** [comment]
Changes already merged.

**Pull request closed without merging**